application/controllers/collection_instrument.py

```diff
@@ -72,7 +72,7 @@ def upload_instrument(self, exercise_id, file, ru_ref=None, classifiers=None, se
         :return 'UPLOAD_SUCCESSFUL' if the upload completed
         """
 
-        log.info('Upload exercise: {}'.format(exercise_id))
+        log.info('Upload exercise: ', exercise_id=exercise_id.format)
```

Comment:
The standard way we do logging in the other services would be 
`log.info('Upload exercise', exercise_id=exercise_id)`
Since everything gets put into json format the colon doesn't really fit here.
Also shouldn't have `.format` here

Reply:
I missed that .format good spot 👍